src/torchcodec/decoders/_core/VideoDecoder.h

```diff
@@ -361,7 +365,8 @@ class VideoDecoder {
   // DECODING APIS AND RELATED UTILS
   // --------------------------------------------------------------------------
 
-  bool canWeAvoidSeeking(int64_t targetPts) const;
+  bool canWeAvoidSeekingVideo(int64_t targetPts) const;
+  bool canWeAvoidSeekingAudio(double desiredPtsSeconds) const;
```

Comment:
Curious that we have to have different logic for both - and that they don't take the same kind of value. (I'm reading the .h first to get a survey of what changed; this is my note to dig into this deeper when I see the implementation.)

Reply:
Revisiting this comment after reading the implementation - I think it may be possible to get them to the same signature, and hide them behind a general dispatch.